src-ghc/Pact/Types/Server.hs

```diff
@@ -78,6 +79,9 @@ data CommandConfig = CommandConfig {
     , _ccEntity :: Maybe EntityName
     , _ccGasLimit :: Maybe Int
     , _ccGasRate :: Maybe Int
+    , _ccChainId :: Maybe Word32
```

Comment:
Don't add to config, we'll use a repl function

Reply:
Removed